library/common/http/client.cc

```diff
@@ -377,20 +401,21 @@ void Client::setDestinationCluster(Http::RequestHeaderMap& headers) {
   auto h2_header = headers.get(H2UpstreamHeader);
   auto network = preferred_network_.load();
   ASSERT(network >= 0 && network < 3, "preferred_network_ must be valid index into cluster array");
+  ASSERT(!alternate, "every time");
```

Comment:
I might be being dense, but I am not sure I understand this assertion?

Reply:
Just leftover from testing. :)